src/cf-forms/src/molecules/form-fields.less

```diff
@@ -101,4 +101,73 @@
             }
         }
     }
+
+    &__select {
+        .a-select {
+            position: relative;
+            margin-top: 0.5em;
```

Comment:
Also, the margin top should probably be done the same as on inputs, because we wouldn't want it added if there was no label inside the field, like if there's a label heading outside the field. That margin would be handled by the label heading itself.